packages/components/src/useScreenSize/isSmallScreen.js

```diff
@@ -0,0 +1,7 @@
+import breakpoints from './breakpoints';
+
+const variants = Object.keys(breakpoints);
+
+const isSmallScreen = screenSize => variants.indexOf(screenSize) <= variants.indexOf('sm');
```

Comment:
Why two `indexOf`? It can be done just by simple object access
```JS
const isSmallScreen = screenSize => (breakpoints?.[screenSize] <= breakpoints.sm);
```